presto-main/src/main/java/io/prestosql/operator/DynamicFilterSourceOperator.java

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
Consider adding some unit-tests to this operator.

Reply:
I've added `TestDynamicFilterSourceOperator` https://github.com/prestosql/presto/pull/1072/commits/52fee8e61755e96d0f4c276b0b7aa62e127a8970
It's very similar to the tests in your PR but it makes use of `DynamicFilterService` and `InMemoryDynamicFilterClientSupplier`